src/pages/HomePage/components/Categories/Categories.jsx

```diff
@@ -1,22 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import styles from './Categories.module.sass'
 import ArrowDropRightFillIcon from 'remixicon-react/ArrowDropRightFillIcon'
 import {useTranslation} from 'react-i18next'
+import { useSelector, useDispatch } from 'react-redux'
+import { getPostApi, setFilter, setCurrentPage } from '../../../../store/slices/ProductSlice'
 
 function Categories() {
 
     const { t } = useTranslation()
 
-    const listCategories = ['Electronic', 'Computer', 'Relux', 'Game', 'Food', 'Decor']
+    const dispatch = useDispatch()
+    const { categoriesList } = useSelector( state => state.products.categories)
+    const filter = useSelector( state => state.products.filter)
+
+    useEffect(() => {
+        dispatch(getPostApi())
+    },[])
```

Comment:
```suggestion
    const { categoriesList } = useSelector(state => state.products.categories)
    const filter = useSelector(state => state.products.filter)

    useEffect(() => {
        dispatch(getPostApi())
    }, [])
```
Chú ý kĩ format code nhé